### PR TITLE
Козлов Дмитрий

### PR DESCRIPTION
**Pull request by RedDime1**
https://codesandbox.io/p/sandbox/letter-qllzst?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cm10jv7u80006356izgdn7a92%2522%252C%2522sizes%2522%253A%255B100%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cm10jv7u70002356iw0yufuxk%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cm10jv7u70003356ilefxk5qq%2522%257D%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cm10jv7u70005356ikvjq0pso%2522%257D%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cm10jv7u70002356iw0yufuxk%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm10jv7u70001356i4k6cm3ue%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Findex.html%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522id%2522%253A%2522cm10jv7u70002356iw0yufuxk%2522%252C%2522activeTabId%2522%253A%2522cm10jv7u70001356i4k6cm3ue%2522%257D%252C%2522cm10jv7u70005356ikvjq0pso%2522%253A%257B%2522id%2522%253A%2522cm10jv7u70005356ikvjq0pso%2522%252C%2522activeTabId%2522%253A%2522cm10kzg2f004b356ixx8a2flc%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm10jv7u70004356if12umdqf%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%257D%252C%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522id%2522%253A%2522cm10jvbu50010356iucmbxnnc%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%252C%257B%2522type%2522%253A%2522DOCS%2522%252C%2522id%2522%253A%2522cm10kzg2f004b356ixx8a2flc%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%252C%2522cm10jv7u70003356ilefxk5qq%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522cm10jv7u70003356ilefxk5qq%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Autobiography" />
```
Comment (mokhov) [inline index.html]: lang="ru" а пишешь по английски

File: index.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        <br />
+        Не надо...
+      </h1>
+      <nav>
+        <h3>Ух ты, навигация:</h3>
+        <ul>
+          <li><a href="#whoami">Кто я?</a></li>
+          <li><a href="#hobby">Зачем я?</a></li>
+          <li><a href="#facts">Интересные факты</a></li>
+          <li><a href="#">Здесь мог быть рик-ролл</a></li>
```
Comment (mokhov) [inline index.html]: Но почему же его нет? :)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Autobiography" />
+    <meta name="author" content="Kozlov Dmitry" />
+    <title>Autobiography</title>
+  </head>
+  <body>
+    <header>
+      <h1>
+        Кто же такой Козлов Дмитрий?
+        <span style="text-decoration: line-through">И с чем его едят?</span>
```
Comment (mokhov) [inline index.html]: Тут семантичнее использовать `<del>` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/del

File: index.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+          <li><a href="#hobby">Зачем я?</a></li>
+          <li><a href="#facts">Интересные факты</a></li>
+          <li><a href="#">Здесь мог быть рик-ролл</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h2>Давайте знакомиться!</h2>
+      <article>
+        <div id="whoami">
```
Comment (mokhov) [inline index.html]: Этот div лишний

File: index.html [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+          <h4>Кто и откуда взялся?</h4>
```
Comment (mokhov) [inline index.html]: Не очень понятно почему после h2 ты пользуешься h4?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+          ／＞　 フ
+　　　　　| 　_　 _|
+　 　　　／`ミ _x 彡
+　　 　 /　　　 　 |
+　　　 /　 ヽ　　 ﾉ
+　／￣|　　 |　|　|
+　| (￣ヽ＿_ヽ_)_)
+　＼二つ
+              </pre
+            >
```
Comment (mokhov) [inline index.html]: А почему уехало на следующую строчку?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+              </caption>
+              <thead>
+                <tr>
+                  <th>Поддаться ностальгии</th>
+                  <th>Отказаться от ностальгии</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Heroes of Might and Magic III</td>
```
Comment (mokhov) [inline index.html:133]: 🔥

**Review by mokhov — COMMENTED** [review]
В целом хорошо, но есть что улучшить

File: index.html (hunk 1 above)
Comment (RedDime1) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (RedDime1) [reply index.html]: Неэтично)
Теперь есть, fixed

File: index.html (hunk 3 above)
Comment (RedDime1) [reply index.html]: fixed

File: index.html (hunk 4 above)
Comment (RedDime1) [reply index.html]: fixed, id привязал к заголовкам

File: index.html (hunk 5 above)
Comment (RedDime1) [reply index.html]: Контраст понравился...
Теперь после h2 следует h3

File: index.html (hunk 6 above)
Comment (RedDime1) [reply index.html]: Не уследил. fixed

**Comment by RedDime1** [comment]
Изменения внесены. По той же ссылке отображается новое состояние

File: index.html [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+        </aside>
+        <p>
+          Зато из-за необычной специфики заданий в таких соревнованиях, кажется,
+          что я слишком много раз видел <samp>Hello world!</samp> на разных
+          языках.
+        </p>
+      </article>
+      <article>
+        <h3 id="facts">Интересные факты обо мне</h3>
+        <ol start="0">
```
Comment (mokhov) [inline index.html:86]: 👍

**Review by mokhov — COMMENTED** [review]
Принято

**Pull request closed without merging**